main.py

```diff
@@ -1,15 +1,17 @@
 from modules import Clients, ServerWithGCI, GCI
 import simpy
 
-SIM_DURATION = 10
+SIM_DURATION = 12
 env = simpy.Environment()
 
-gci = GCI(env)
-server = ServerWithGCI(env, gci)
+server = ServerWithGCI(env)
 requests = list()
 clients = Clients(env, server, requests)
 
 env.run(until=SIM_DURATION)
 
+print("heap %.10f" % server.heap.level)
 print("remaining request in queue: %i" % len(server.queue.items))
```

Comment:
Nit: s/request/requests (requests is plural)

Reply:
My bad. I'll fix it.

Reply:
Done, @danielfireman could you review?